scripts/MultiPlotting/navigation_toolbar.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import (absolute_import, division, print_function)
+from qtpy import QtGui
+from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
+
+
+class myToolbar(NavigationToolbar):
+    # only display the buttons we need
+    toolitems = [tool for tool in NavigationToolbar.toolitems if
+                 tool[0] in ("Home", "Save", "Pan", "Zoom")]
+
+    def __init__(self, *args, **kwargs):
+        super(myToolbar, self).__init__(*args, **kwargs)
+        self.layout().takeAt(5)  # or more than 1 if you have more buttons
+        pm = QtGui.QPixmap()
```

Comment:
Does this and the line below do anything?

Reply:
that should be commented out too